{
  "model": {
    "shape": "(1,1,28,28)",
    "bounds" : "[(0,1)]",
    "layers": [  
      {
        "type": "Conv2D",
        "weights": "",
        "bias": "",
        "nodes": "36",
        "kernel_size": "(2,2)",
        "padding": "true"
      },
      {
        "type": "BatchNormalization"
      },
      {
        "type": "relu"
      },
      {
        "type": "Conv2D",
        "weights": "",
        "bias": "",
        "nodes": "18",
        "kernel_size": "(2,2)",
        "padding": "true"
      },
      {
        "type": "BatchNormalization"
      },
      {
        "type": "relu"
      },
      {
        "type": "Conv2D",
        "weights": "",
        "bias": "",
        "nodes": "9",
        "kernel_size": "(2,2)",
        "padding": "true"
      },
      {
        "type": "BatchNormalization"
      },
      {
        "type": "relu"
      },
      {
        "type": "Flatten"
      },      
      {
        "type": "linear",
        "weights": "",
        "bias": "",
        "nodes": "345",
        "func": "relu"
      },
      {
        "type": "softmax"
      }
    ]
  },
  "source":"https://stackoverflow.com/questions/56624408/typeerror-and-valueerror-while-training-model-with-keras"
}
